nimble/controller/src/ble_ll_conn.c

```diff
@@ -2576,6 +2598,12 @@ ble_ll_conn_next_event(struct ble_ll_conn_sm *connsm)
         ble_ll_conn_itvl_to_ticks(connsm->conn_itvl, &connsm->conn_itvl_ticks,
                                   &connsm->conn_itvl_usecs);
 
+        if (connsm->conn_param_req.handle != 0) {
```

Comment:
handle does not seem to be reset anywhere so better reset it after procedure is completed